packages/ingest/granule.js

```diff
@@ -533,6 +533,7 @@ class Granule {
         url_path: this.getUrlPath(file),
         bucket
       });
+    if (exists) stagedFile.duplicate_found = true;
```

Comment:
`exists` is a pretty generic name.  Could something along the lines of `s3ObjectAlreadyExists` be used instead?

Reply:
Heh, noticed when searching for this variable name that you're disagreeing with [yourself](https://github.com/nasa/cumulus/blame/b58558f7056194d2bb4438f6f25898451c01a4d9/packages/ingest/granule.js#L131).
You're right, though, I've updated the names.